File: comparison/fibR.c
#include <stdio.h>

int fibR(int n) {
  if (n == 0) {
    return 0;
  } else if (n == 1) {
    return 1;
  } else {
    return fibR(n - 1) + fibR(n - 2);
  }
}

int main() {
  for (int i = 30; i < 40; ++i) {
    printf("%d\n", i);
    printf("%d\n", fibR(i));
  }
}

File: comparison/fibR.py
import time
import sys


def fibR(n):
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fibR(n-1) + fibR(n-2)


t = time.time()

for i in range(30, 40):
    print(i)
    print(fibR(i))

delta_t = time.time() - t
print(delta_t, file=sys.stderr)
